importMariaDBdmpFile/config.ini

```diff
@@ -0,0 +1,25 @@
+[remote]
+# hostname of remote host from which to fetch the dump file
+host=somehost.ral.ac.uk
+# remote user. Note ssh access must be without password i.e. by key
+user=dbsync
+# path on the remote host from which to fetch the dump file
+path=/path/to/db/dump
+
+[local]
+# local 'working' directory to stage and/or inflate the dump file into.
+workDir=/tmp
+# Options file containing the password with the format -
+# [client]
+# password=myBadPassword
+mysqlOptions=/path/to/password/file
+# local directory into which successfully installed dumps are archived
+# a _goc5dump
+archiveDir=/root/importMariaDBdmpFile/lastImportedDmpFile
+# time format string to generate archived filename
+format='_goc5dump_%y-%m-%d_%H:%M_%S'
+
+[logs]
+# set to 'info' for step-by-step summary
+# only 'error' or 'info' are implemented
+level=error
```

Comment:
What's the output like when the logging level is `error`? The current monitoring assumes no output on a successful run.

Reply:
My reading is that the current monitoring requires "completed ok" as the final line of output from a successful run. Any other condition is a failure. 
I am re-working the logging to allow specification of a log file to which info/debug/error output is written while "completed ok" or an error message goes to stdout.

Reply:
so "completed ok" currently goes to the syslog via the `logger` command, whereas [the monitoring](https://github.com/GOCDB/gocdb-failover-scripts/pull/9/files#diff-59fc84ab2921cdbfec67ff8af73dc707ca13c78c80a8c2dbb23ebe287f19154d) checks the log produced by the cron job that runs the import process hourly and assumes the underlying `1_runDbUpdate.sh` produces no output to stdout or stderr on a successful import.

You could have the logging report at the debug level rather than info - so if something is going wrong with the process one can switch the more verbose mode on.

Reply:
:lightbulb: 1_runDbUpdate.sh has an internal bash function 'logger' which (appears to me) is hardwired to output to a local file, so not to syslog. How does the output end up in syslog? 

In any case, I've changed it so that stdout should be empty and "completed ok", or error message, will go in the specified log file, with return status as appropriate.

Reply:
ah yes, you're right - I mistook `logger` for the [pre defined function](https://man7.org/linux/man-pages/man1/logger.1.html) and not one that was defined in the script itself. And I've only just discovered updateLog.txt is even a thing...

error messages should go to stderr as well, I'm not convinced of the need to have two log files - one for the script and one for the cron.